### ISSUE TITLE
Tensorflow version

### ISSUE THREAD

**Issue by RegisWu** [HuiDingUMD/ExprGAN#6]
Hi, may I know which version of the tensorflow you use for the implementation?

Thank you.

**Comment by voa18105** [comment]
@RegisWu did you find out? 1.2?
1.8 obvisously does not work, I've started re-writing functions with old syntax, but at some point i've stucked completely...

**Comment by 13293824182** [comment]
@RegisWu @voa18105  Have you solved this problem?

**Comment by voa18105** [comment]
@13293824182 Yes, I've managed to pass the quest of repeating original train/test completely and I can tell you its a waste of time.

**Comment by 13293824182** [comment]
may I know which version of the tensorflow you use for the implementation?can you give me more guidance ? thank you
my email is 13293824182@163.com

**Comment by 13293824182** [comment]
@voa18105

**Comment by voa18105** [comment]
@13293824182 I use 1.8, but I had to edit codes to adapt something... I don't remember already. What I do remember is that this code overfits badly with the dataset. Check this https://github.com/HuiDingUMD/ExprGAN/issues/2

**Comment by 13293824182** [comment]
can you share your code and data with me? @voa18105
